apps/minifront/src/components/extension-not-connected.tsx

```diff
@@ -62,15 +56,15 @@ export const ExtensionNotConnected = () => {
     <>
       <HeadTag />
       <Toaster />
-      <SplashPage title='Welcome to Penumbra'>
+      <SplashPage title='Connect to Penumbra'>
         <div className='flex items-center justify-between gap-[1em] text-lg'>
           <div>To get started, connect the Penumbra Chrome extension.</div>
           {!result ? (
-            <Button variant='gradient' onClick={() => void request()} className='px-4'>
```

Comment:
Were there design changes? Can you share screenshots on changes?

Reply:
the error pages all had different font sizes and such. i just made styling between all equivalent.